src/sentry/api/endpoints/organization_user_issues_search.py

```diff
@@ -30,18 +30,21 @@ def get(self, request, organization):
             ).values_list('id', flat=True)[:1000]
         )
 
-        event_users = EventUser.objects.filter(
+        event_users = list(EventUser.objects.filter(
             email=email,
             project_id__in=project_ids,
-        )[:1000]
-
-        project_ids = list(set([e.project_id for e in event_users]))
-
-        group_ids = tagstore.get_group_ids_for_users(project_ids, event_users, limit=limit)
-
-        groups = Group.objects.filter(
-            id__in=group_ids,
-        ).order_by('-last_seen')[:limit]
+        )[:1000])
+
+        if event_users:
+            groups = Group.objects.filter(
+                id__in=tagstore.get_group_ids_for_users(
+                    project_ids=list(set([e.project_id for e in event_users])),
+                    event_users=event_users,
+                    limit=limit,
+                ),
+            ).order_by('-last_seen')[:limit]
```

Comment:
I know collapsing this into a single assignment probably drives some people crazy, but I did this to avoid the likelihood of someone later accidentally using these variables out of this `if` block or causing shadowing problems down the line.